staging/src/k8s.io/component-base/version/version.go

```diff
@@ -134,15 +136,15 @@ func (m *effectiveVersion) Validate() []error {
 	emulationVersion := m.emulationVersion.Load()
 	minCompatibilityVersion := m.minCompatibilityVersion.Load()
 
-	// emulationVersion can only be 1.{binaryMinor-1}...1.{binaryMinor}.
+	// emulationVersion can only be 1.{binaryMinor-3}...1.{binaryMinor}
 	maxEmuVer := binaryVersion
-	minEmuVer := binaryVersion.SubtractMinor(1)
+	minEmuVer := binaryVersion.SubtractMinor(3)
 	if emulationVersion.GreaterThan(maxEmuVer) || emulationVersion.LessThan(minEmuVer) {
 		errs = append(errs, fmt.Errorf("emulation version %s is not between [%s, %s]", emulationVersion.String(), minEmuVer.String(), maxEmuVer.String()))
 	}
-	// minCompatibilityVersion can only be 1.{binaryMinor-1} for alpha.
-	maxCompVer := binaryVersion.SubtractMinor(1)
-	minCompVer := binaryVersion.SubtractMinor(1)
+	// minCompatibilityVersion can only be 1.{binaryMinor-3} to 1.{binaryMinor}
+	maxCompVer := binaryVersion
+	minCompVer := binaryVersion.SubtractMinor(3)
```

Comment:
```suggestion
	maxCompVer := emulationVersion
	minCompVer := binaryVersion.SubtractMinor(4)
```

Reply:
When `emulationVersion  = binaryVersion - 3`, we still need to support `minCompatibilityVersion =emulationVersion-1`

Reply:
Can you also add 
```diff

func (m *effectiveVersion) SetEmulationVersion(emulationVersion *version.Version) {
	m.emulationVersion.Store(majorMinor(emulationVersion))
+.     // set the default minCompatibilityVersion to be emulationVersion - 1
+      m.minCompatibilityVersion.Store(majorMinor(minCompatibilityVersion.SubtractMinor(1)))
}

+ // SetMinCompatibilityVersion should be called after SetEmulationVersion
func (m *effectiveVersion) SetMinCompatibilityVersion(minCompatibilityVersion *version.Version) {
	m.minCompatibilityVersion.Store(majorMinor(minCompatibilityVersion))
}
?
```

Reply:
Fixed